blog/views.py

```diff
@@ -1,13 +1,15 @@
 from django.contrib.auth.models import User
 from django.db.models import Q
 from django.utils import timezone
-from rest_framework import viewsets, permissions
+from rest_framework import permissions, viewsets
 from rest_framework.decorators import action
+from rest_framework.generics import get_object_or_404
 from rest_framework.response import Response
 
-from blog.models import Blog, Comment, UserVote
+from blog.models import Blog, Comment, UserVote, Profile
 from blog.permissions import IsOwnerOrReadOnly
-from blog.serializers import BlogSerializer, UserSerializer, CommentSerializer, VoteSerializer
+from blog.serializers import (BlogSerializer, CommentSerializer,
```

Comment:
Make these group of imports properly indented.

Reply:
Done.